app/serializers/framework_serializer.rb

```diff
@@ -2,14 +2,18 @@
 
 class FrameworkSerializer
   include JSONAPI::Serializer
+  include JSONAPI::ConditionalRelationships
 
   set_type :frameworks
 
-  has_many :questions, &:framework_questions
+  has_many_if_included :questions, &:framework_questions
 
   attributes :name, :version
 
   SUPPORTED_RELATIONSHIPS = %w[
     questions
+    questions.descendants.**
```

Comment:
since there might be a bit of rejig to serializers maybe we can include all flags on the question level to avoid having to call descendants as well? (unless this still causes an n+1 after rejigg)